packages/gateway/src/graphql/config.ts

```diff
@@ -105,6 +116,10 @@ export const getApolloConfig = (): Config => {
     typeDefs,
     resolvers,
     introspection: true,
+    dataSources: () => ({
+      locationsAPI: new LocationsAPI(),
+      practitionerRoleAPI: new PractitionerRoleAPI()
+    }),
     context: async ({ request, h }) => {
```

Comment:
```ts
context: async ({ request, h }): Context['headers'] => {
```

Reply:
Like above, done in https://github.com/opencrvs/opencrvs-core/pull/4706/commits/270763159f7c733bf6296823867ad7560ffc0b0e